symplyphysics/symbols/classical_mechanics.py

```diff
@@ -362,3 +372,30 @@
 **Angular momentum**, sometimes called **rotational momentum**, is the rotational analog of linear
 :symbols:`momentum`.
 """
+
+latitude = SymbolNew("phi", angle_type, display_latex="\\phi")
+"""
+**Latitude** is a coordinate that specifies the north-south position of a point on the surface of
+the Earth or another celestial body. Its value ranges from :math:`-90^\circ` at the south pole
+to :math:`90^\circ` at the north pole, with :math:`0^\circ` at the Equator.
+"""
+
+longitude = SymbolNew("lambda", angle_type, display_latex="\\lambda")
+"""
+**Longitude** is a coordinate that specifies the east-west position of a point on the
+surface of the Earth, or another celestial body. The prime meridian defines :math:`0^\circ`
+longitude; positive longitudes are east of the prime meridian, and the negative ones are west.
+"""
+
+sector_speed = SymbolNew("sigma", units.area / units.time, display_latex="\\sigma")
+"""
+**Areal speed**, also called **sector speed** or **sectorial speed**, is a quantity that indicated
+the rate of change at which :symbols:`area` is swept out by a particle as it moves along a curve.
```

Comment:
Not clear how this area is calculated. Elaborate on the description or give the link to the detailed description.